### PR TITLE
Support Batfish open source as well

### PR DESCRIPTION
**Pull request by ratulm**
Made OSS the default.

### CODE REVIEW

File: bfe_upload_snapshot.py [hunk 1]
```diff
@@ -34,14 +34,17 @@ def main(bf: Session, bf_network: str, snapshot_dir: str) -> None:
...
+    bfe_port = settings.get('BFE_PORT', 443)
     bf_network = settings.get('BF_NETWORK', None)
 
     if bf_host is None:
         raise Exception(f"BF_HOST is not set in {args.settings}")
     if bf_network is None:
         raise Exception(f"BF_NETWORK is not set in {args.settings}")
 
-    bf = Session(host=bf_host, port=bf_port, access_token=args.access_token)
+    print(bf_enterprise)
```
Comment (saparikh) [inline bfe_upload_snapshot.py:46]: I am presuming this is a left over print statement that isn't needed?

Comment (ratulm) [reply bfe_upload_snapshot.py]: yes, fixed.

File: snapshot_network.sh [hunk 2]
```diff
@@ -15,11 +15,6 @@ if [[ -z ${BF_COLLECTOR_PASSWORD+x} ]]; then
     exit 1
 fi
 
-if [[ -z ${BF_ACCESS_TOKEN+x} ]]; then
-    echo "Environment variable BF_ACCESS_TOKEN is not set"
-    exit 1
-fi
-
 if [[ -z ${2+x} ]]; then
     echo "Usage: process_collection.sh <inventory file> <batfish settings file> <optional: collection directory>)"
```
Comment (saparikh) [inline snapshot_network.sh:19]: Comment references `process_collection.sh` but script name is `snapshot_network.sh`

Comment (ratulm) [reply snapshot_network.sh]: old bug. fixed.

**Pull request merged**